libs/sdk-backend-tiger/src/backend/workspace/measures/index.ts

```diff
@@ -201,4 +202,20 @@ export class TigerWorkspaceMeasures implements IWorkspaceMeasuresService {
             };
         });
     };
+
+    public getMeasures = async (measureRefs: ObjRef[]): Promise<IMeasureMetadataObject[]> => {
+        return this.authCall(async (client) => {
+            const allMetrics = await loadMetrics(client, this.workspace);
+
+            return allMetrics.filter((metric) =>
+                measureRefs.find((metricRef) => areObjRefsEqual(metricRef, metric.ref)),
+            );
+        });
+    };
+}
+
+function loadMetrics(client: ITigerClient, workspaceId: string): Promise<IMeasureMetadataObject[]> {
+    return MetadataUtilities.getAllPagesOf(client, client.entities.getAllEntitiesMetrics, { workspaceId })
```

Comment:
Please try using the RSQL to filter on the backend if the requested number of metrics is small enough to fit the query string.